readme.txt

```diff
@@ -221,6 +221,7 @@ Still not happy? Shoot us an email to support@theeventscalendar.com or tweet to
 * Feature - Added new `tribe_events_set_month_view_events_from_cache` action to make it easier to listen for when Month View events are retrieved from the Month View cache [116124]
 * Fix - Importing CSV files with whitespace at the beginning of the rows won't remove headers and skip columns on Preview [117236]
 * Fix - Prevent JavaScript error in Month View when the Date filter is removed from the Tribe Bar (thank you to @tyrann0us on GitHub for submitting this fix!) [116995]
+* Fix - Make sure Event Pro update correctly when using an outdated version
```

Comment:
Minor thing, but, I think given the audience we should refer to `Events Calendar PRO` vs `Event Pro`, here. Maybe (?) the following:

_* Fix - Facilitate automated updates for Events Calendar PRO even if it's requirements have not been met_

Reply:
Thanks Barry! ❤️